app/src/app/components/specmath-navbar/specmath-navbar.component.html

```diff
@@ -0,0 +1,17 @@
+<!-- Copyright 2020 Google LLC
```

Comment:
I think for this component (and the others to follow) you can drop the spec-math prefix on the filename.

Reply:
Done 👍